### ISSUE TITLE
[QST] Spark-SQL support is available ?

### ISSUE THREAD

**Issue by Manoj-red-hat** [NVIDIA/spark-rapids#5234]
Most of the articles and blog always talk about SQL via dataframes approach, so just wondering **Can I run SQL queries via spark-rapids[Spark-SQL shell] ?**

**Comment by SoumyaB57** [comment]
Yes we can do that just add the jar files and plugin when running spark-shell script

**Comment by Manoj-red-hat** [comment]
@SoumyaB57 Can I run that on @nvidia geforce gtx 1050 for learning purpose ?

**Comment by jlowe** [comment]
@Manoj-red-hat yes, it should run on a GTX 1050 because that uses the Pascal architecture.  The RAPIDS Accelerator support the same GPUs as RAPIDS, see the [RAPIDS Requirements page](https://rapids.ai/start.html#prerequisites) for details.

By the way, for future questions it would be great to use our new [Discussions section](https://github.com/NVIDIA/spark-rapids/discussions).
